src/main/java/net/tropicraft/client/gui/GuiTropicsLoading.java

```diff
@@ -0,0 +1,312 @@
+package net.tropicraft.client.gui;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Random;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.commons.lang3.tuple.Triple;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.gui.ScaledResolution;
+import net.minecraft.client.network.NetHandlerPlayClient;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.OpenGlHelper;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.client.renderer.VertexBuffer;
+import net.minecraft.client.renderer.entity.RenderManager;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityList;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.item.ItemStack;
+import net.minecraft.network.play.client.CPacketKeepAlive;
+import net.minecraft.util.text.translation.I18n;
+import net.minecraftforge.fml.client.FMLClientHandler;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+import net.tropicraft.core.client.TropicraftRenderUtils;
+import net.tropicraft.core.common.entity.passive.EntityKoaHunter;
+import net.tropicraft.core.common.entity.underdasea.atlantoku.EntityTropicraftWaterBase;
+import net.tropicraft.core.registry.BlockRegistry;
+import net.tropicraft.core.registry.ItemRegistry;
+import net.tropicraft.core.registry.TropicraftRegistry;
+
+@SideOnly(Side.CLIENT)
+public class GuiTropicsLoading extends GuiScreen {
+
+	private static final String[] CATEGORIES = { "misc", "knowledge", "hint" };
+	private static final String[] BACKGROUNDS = { "loading_bg1", "loading_bg2", "loading_bg3" };
+
+	private static final String[] MOBS_WATER = { "marlin", "dolphin", "hammerhead", "turtle", "seahorse", "failgull" };
+	private static final String[] MOBS_LAND = { "iguana", "failgull", "redfrog", "bluefrog", "greenfrog", "yellowfrog",
+			"monkey" };
+	private static final String[] MOBS_VILLAGE = { "koa", "ashen", "tropicreeper", "tropiskelly" };
```

Comment:
I see these becoming problematic in the future if we change one of the registry names. This doesn't need to be done now, but we should probably move those registry names to an enum or at least string constants that can be referenced anywhere. Just a task worthy of going on the backlog for now.